examples/funloc/analysis_fun.py

```diff
@@ -66,7 +66,7 @@
                     [1, 1, 2],  # EOG
                     [0, 0, 0]]  # Continuous (from ERM)
 params.cov_method = 'shrunk'  # cleaner noise covariance regularization
-
+params.sss_type = 'python'  # python | maxfilter for choosing SSS applied using either Maxfilter or mne-python
```

Comment:
this shouldn't break anything because the `sss_type` parameter should already exist